src/meta/const_value.go

```diff
@@ -203,6 +203,14 @@ func (c ConstValue) String() string {
 	return fmt.Sprintf("%s(%v)", c.Type, c.Value)
 }
 
+func (c ConstValue) StringValue() string {
```

Comment:
If we move this method to some util function in linter package, I can approve this PR.